package.json

```diff
@@ -60,6 +60,7 @@
     "react-dom": "^15.4.2",
     "react-hot-loader": "^3.0.0-beta.6",
     "react-redux": "^5.0.2",
+    "react-router-dom": "^4.0.0-beta.7",
```

Comment:
this was stored in the `devDependencies` so now it's in both places, can you remove it from devDeps, or move it back there?

I feel like we could probably move a bunch of the 'frontend only' depends to the devDependencies, our publish script can include the `dist` bundle so the backend won't need to compile it, or we can move everything into dependencies that doesn't involve tests only basically.